main.tf

```diff
@@ -153,3 +153,31 @@ resource "google_cloudfunctions2_function_iam_member" "developers" {
     google_cloudfunctions2_function.function
   ]
 }
+
+// IAM for invoking HTTP functions (roles/run.invoker)
+resource "google_cloud_run_service_iam_member" "invokers" {
```

Comment:
@bharathkkb Not necessarily. I thought this could affecting existing users and left both of them. [Previous comment](https://github.com/GoogleCloudPlatform/terraform-google-cloud-functions/pull/88#issuecomment-1996402339).